index.md

```diff
@@ -63,6 +63,8 @@ Note: Currently Tizen/TAU specific extensions are under development.
 ## Documentation
 
 * [TAU API Reference](https://developer.tizen.org/dev-guide/5.0.0/org.tizen.web.apireference/html/ui_fw_api/ui_fw_api_cover.htm)
+* [Examples of TAU UIComponents - mobile](examples/mobile/UIComponents/index.html)
+* [Examples of TAU UIComponents - wearable](examples/wearable/UIComponents/index.html)
```

Comment:
This may works but the TAU in UIComponents is linked by symlink to /dist folder.

Reply:
Right. I'll add link to examples together with TAU lib